MREUnityRuntime/MREUnityRuntimeLib/Messaging/Payloads/PayloadTypeRegistry.cs

```diff
@@ -43,7 +43,7 @@ namespace MixedRealityExtension.Messaging.Payloads
     [PayloadType(typeof(SetAnimationState), "set-animation-state")]
     [PayloadType(typeof(SetAuthoritative), "set-authoritative")]
     [PayloadType(typeof(SetBehavior), "set-behavior")]
-    [PayloadType(typeof(SetSoundState), "set-sound-state")]
```

Comment:
I think we're fine with breaking back compatibility here, but want to make sure we're in agreement that this is a breaking change.

Reply:
I will call this out in SDK channel.